File: Testes/Primeiro Projeto/src/teste/Teste.java
package teste;
public class Teste {
    public static void main(String[] args) throws Exception {
        Caneta caneta1 = new Caneta();
        caneta1.modelo = "Cis Spiro";
        caneta1.cor = "Azul e Branco";
        caneta1.ponta = 0.7f;
        caneta1.carga = 70;
        caneta1.destampar();
        caneta1.rabiscar();
        caneta1.tampar();
        caneta1.status();
    }
}

File: C++/Teste/Teste.cpp
#include <iostream>
#include "Caneta.h"
#include <clocale>
#include <cwchar>

using namespace std;

int main() {
    setlocale(LC_ALL, "Portuguese-Brazilian");

    Caneta caneta1 = Caneta("Cis", "Azul e Branco", 0.7);
    caneta1.destampar();
    caneta1.rabiscar();
    caneta1.tampar();
    caneta1.status();

    return 0;
}